### PR TITLE
Use pyproject toml instead of setup.py 

### PR DESCRIPTION
**Pull request by h-mayorquin**
Using the standard file for metadata.

### CODE REVIEW

**Review by pauladkisson — COMMENTED** [review]
This looks good, thanks @h-mayorquin!

Could you update the installation page in the documentation?

**Comment by h-mayorquin** [comment]
OK, I also added the changelog and docs on the pyproject.toml so it gets displayed on pipy.

File: pyproject.toml [hunk 1]
```diff
@@ -1,3 +1,86 @@
...
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Operating System :: POSIX :: Linux",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: MacOS",
+    "License :: OSI Approved :: BSD License",
+]
+urls = { Homepage =  }
```
Comment (pauladkisson) [inline pyproject.toml:32]:
```suggestion
```

Comment (h-mayorquin) [reply pyproject.toml]: Done, also wrong order I think. My bad.

**Comment by pauladkisson** [comment]
Nit: Can you put the urls after optional dependencies? I feel like the dependencies and optional dependencies should go together.

**Comment by h-mayorquin** [comment]
Feel free to re-format this as you prefer actually. I have no strong preference : )

**Comment by pauladkisson** [comment]
> Feel free to re-format this as you prefer actually. I have no strong preference : )

My perfectionist/OCD tendencies kicking in fs here 😂😅

**Review by pauladkisson — APPROVED** [review]

**Comment by h-mayorquin** [comment]
> > Feel free to re-format this as you prefer actually. I have no strong preference : )
> 
> My perfectionist/OCD tendencies kicking in fs here 😂😅

If you can make it more comfortable to work with for no cost then that's great!

**Pull request merged**